packages/sync/src/links/DebounceLink.ts

```diff
@@ -0,0 +1,176 @@
+import {
```

Comment:
If this is used for squashing then -1. Debouncing implementation has inside it's own queue which will complicate testing and development of this functionality. We are kinda replicating the code. Original code for debouncing implemented in spike will be more specialized and we can integrate that carefully as part of the link-queue interface.